src/fontstash.h

```diff
@@ -1041,6 +1058,18 @@ static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned in
 	// Could not find glyph, create it.
 	scale = fons__tt_getPixelHeightScale(&font->font, size);
 	g = fons__tt_getGlyphIndex(&font->font, codepoint);
+	//If tofu glyph fallback to other fonts
+	if(g == 0)
+	{
+		for (i=0; i<font->nfallbacks; ++i)
+		{
+			glyph = fons__getGlyph(stash,stash->fonts[font->fallbacks[i]],codepoint,isize,iblur);
+			if(glyph->index != 0)
```

Comment:
- use local varible for fallback glyph, in case fallback is not found the code below will modify glyph
- glyph can be null